static/src/javascripts/projects/commercial/modules/prebid/prebid.js

```diff
@@ -185,6 +199,24 @@ const initialise = (window: {
             ],
         };
     }
+
+    if (isInVariantSynchronous(commercialPrebidSize, 'variant')) {
+        // Adjust slot size when prebid ad loads
+        window.pbjs.onEvent('bidWon', data => {
```

Comment:
Is this event fired by prebid.js? If so, is there anything ensuring the type remains accurate?

```
type PbjsEventData = {
    width: number,
    height: number,
    adUnitCode: string,
};
```

Pretty unlikely these fields will change so maybe it's not worth worrying about? What happens to the match on line 206 if the fields don't exist at runtime?

Reply:
> What happens to the match on line 206 if the fields don't exist at runtime

Fair point, I've made an assumption they will exist based on what I've seen at runtime, but the prebid documentation doesn't give any defined type/interface for `data` so it'd be prudent to guard against these values not being present. I can update so if they're unavailable then `advert.hasPrebidSize` will remain `false`, and the size from Google Ad Manager will be used.